controllers/api/v1/videoAnalysis.js

```diff
@@ -0,0 +1,189 @@
+const VideoAnalysisModel = require('../../../models/videoAnalysis');
```

Comment:
all the file names should be all small separated by - between words thats the convention we follow today. rename videoAnalysis to video-analysis.js